dataclients/kubernetes/kube.go

```diff
@@ -171,6 +172,9 @@ type Options struct {
 
 	// KubernetesEastWestDomain sets the DNS domain to be used for east west traffic, defaults to "skipper.cluster.local"
 	KubernetesEastWestDomain string
+
+	// DefaultFiltersDir enables default filters mechanism and sets the location of the default filters
```

Comment:
I miss something like: default filters will be applied to all routes

Reply:
Agreed. May be we should also add the term target service to make the context clearer.